src/views/dependencyDataProvider.ts

```diff
@@ -124,6 +124,14 @@ export class DependencyDataProvider implements TreeDataProvider<ExplorerNode> {
         const children = (!this._rootItems || !element) ?
             await this.getRootNodes() : await element.getChildren();
 
+            if (children && element instanceof ContainerNode) {
```

Comment:
@jdneo We might want to leave this check, as without it, it will sort everything always.
![image](https://github.com/user-attachments/assets/26c42682-b1b2-4665-b70f-1ee741d3e356)

Reply:
ok, makes sense.